sn_node_manager/src/main.rs

```diff
@@ -108,18 +108,23 @@ pub enum SubCmd {
         log_dir_path: Option<PathBuf>,
         #[command(flatten)]
         peers: PeersArgs,
-        /// Specify an Ipv4Addr for the node's RPC service to run on. The ports are assigned automatically.
-        ///
-        /// If not used, the localhost will be used with a random port.
-        /// Specify a port for the node to run on.
-        ///
-        /// If not used, a port will be selected at random.
+        /// Specify a port for the node to run on. If not used, a port will be selected at random.
         ///
         /// This option only applies when a single service is being added.
         #[clap(long)]
         port: Option<u16>,
         #[clap(long)]
+        /// Specify an Ipv4Addr for the node's RPC service to run on. This is useful if you want to expose the
+        /// RPC server outside. The ports are assigned automatically.
+        ///
+        /// If not set, the RPC server is run locally.
         rpc_address: Option<Ipv4Addr>,
+        /// Provide the environmental variable that is set for the safenode service.
+        ///
+        /// This is useful to set the safenode's log levels. Each variable should be comma separated without any space.
+        /// Example usage `--env SN_LOG=all,RUST_LOG=libp2p=debug`
```

Comment:
I would suggest changing this to:
```
This is useful to set the safenode's log levels. Each variable should be comma separated without any space.

Example: --env SN_LOG=all,RUST_LOG=libp2p=debug
```

With a line break between the two lines, and the backticks removed. The backticks don't get parsed in the terminal UI, so I generally have a preference not to use them.